api/services/StatisticsMassifService.js

```diff
@@ -0,0 +1,231 @@
+const CREATE_MATERIALIZED_VIEW = `
+  CREATE MATERIALIZED VIEW v_massif_info AS
```

Comment:
move to .sql file ?

Reply:
same as above

Reply:
I prefer not to introduce source code which will be removed later. Indeed, each PR should be run in production when merged. It would not be the case with this one in the current conditions ;)
If the PR is not ready for production because it miss the cron part you could maybe set this PR as a draft for now ?